tests/real_tests.rs

```diff
@@ -61,9 +62,16 @@ impl Drop for RealTestDev {
     }
 }
 
-/// Set up count devices from configuration file.
-/// Wipe first MiB on each device.
-fn get_devices(count: u8) -> Option<Vec<RealTestDev>> {
+#[allow(dead_code)]
```

Comment:
Please move the allow so that it is just above the Range constructor definition.

Reply:
ok